packages/slate/src/changes/with-schema.js

```diff
@@ -67,42 +66,26 @@ function normalizeNodeAndChildren(change, node, schema) {
     normalizeNode(change, node, schema)
     return
   }
+  const checkedKeys = []
+  let child = node.nodes.first()
+
+  let nodePath = change.value.document.getPath(node.key)
```

Comment:
Can you change this variable to be just `path`?

Reply:
Yes, I will change it.